store/tikv/gcworker/gc_worker.go

```diff
@@ -625,6 +627,17 @@ func (w *GCWorker) sendUnsafeDestroyRangeRequest(ctx context.Context, startKey [
 
 	wg.Wait()
 
+	// Notify all affected regions in the range that UnsafeDestroyRange occurs.
+	notifyTask := tikv.NewNotifyDeleteRangeTask(w.store, startKey, endKey, concurrency)
+	err = notifyTask.Execute(ctx)
+	if err != nil {
```

Comment:
What if `UnsafeDestroyRange` sent failed or `DeleteRangeNotify` sent failed?

Reply:
It should log the error and continue working. Also see #10743 .